### DISCUSSION TITLE
crouton

### DISCUSSION THREAD

**Discussion by Jdiaz574** [dnschneid/crouton#4415] [Q&A]
hey i was wanting to ask is kali linux supported with this method?

**Comment by DennisLfromGA** [comment]
The following kali releases are available in crouton:
```
Recognized kali releases:
    moto! kali! sana! kali-rolling
```
You can see all of the releases by running: `crouton -r list`

I think some have had success with kali-rolling, check https://github.com/dnschneid/crouton/issues for issues, fixes, etc.

*Hope this helps,*
-**DennisLfromGA**
